_posts/2017-11-22-Rust-1.22.md

```diff
@@ -6,7 +6,7 @@ author: The Rust Core Team
 
 The Rust team is happy to announce *two* new versions of Rust, 1.22.0 and
 1.22.1. Rust is a systems programming language focused on safety, speed, and
-concurrency.
+concurrency. Downloads and installation instructions can be found [here](https://www.rust-lang.org/en-US/install.html).
```

Comment:
This is already linked here https://github.com/rust-lang/blog.rust-lang.org/pull/216/files#diff-44d4508ca2fb1f39ac5221d60337e605R25

which is normally the next thing, this announcement happened to have the `>` in the middle.